RFC-676-Contributor-Council-Charter.md

```diff
@@ -0,0 +1,221 @@
+# CDK Contributor Council Charter (RFC) — November, 2024
+
+## Purpose of the Council
+
+To improve visibility into CDK, increase feedback opportunities, and build a stronger, more inclusive community, the AWS CDK team proposes forming a
+Contributor Council.
+
+*Original Author(s)*: [@haubles](https://github.com/haubles), [@billfine](https://github.com/billfine)
+
+*Tracking Issue*: https://github.com/aws/aws-cdk-rfcs/issues/676
+
+## Principles guiding this RFC
+
+1. **Maintain AWS supportability.** In order for AWS to provide on-call resources tasked with resolving high-impact bugs or issues in CDK for
+customers and the community, AWS must ensure that its standards for processes—for example, security reviews—are met. Therefore, AWS controls
+the CDK release process.
+2. **Increase community influence**. The community’s input and active engagement is essential for the long-term health and success of the CDK
+project. Therefore, AWS is committed to creating more robust and clearly-defined channels for soliciting community feedback, and acting on that
+feedback. We are committed to increasing opportunities and building new processes for accepting more community contributions to the codebase.
```

Comment:
The community contributions could be placed in an alternative library, that customers, ( i'm thinking of enteprise customers who pay for support ) would know that is not AWS written code.

Reply:
That is not how OSS works. What you are advocating for is a source-available product instead.

Reply:
It might be OpenSource, but AWS charges money for support.  If it can't demonstrate that it can support the code, then the answer is to say, "heres some possibly other useful stuff, that we've looked at, and think is ok, but its excluded from what we can provide support for".   You cnat' just change the rules of what is being paid for already.